perfkitbenchmarker/scripts/database_scripts/launch_driver.sh

```diff
@@ -42,12 +42,20 @@ fi
 
 cloud_storage_bucket=$1
 per_second_graph=$2
-
+sysbench_run_seconds=1200
+thread_count_list=1,2,4,8,16,32,64,128,256,512
+mysql_svc_db_instance_cores="16"
+mysql_svc_oltp_table_size="12000000"
+mysql_svc_oltp_tables_count="100"
+mysql_instance_storage_size=1000
+prepare_provision=provision,prepare
+run=run
+cleanup_teardown=cleanup,teardown
 
 while true
   do
   # provision, prepare phase of mysql_service
-  run_uri=$(python perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py --run_stage=provision,prepare --mysql_svc_db_instance_cores="16" --mysql_svc_oltp_table_size="12000000" --mysql_svc_oltp_tables_count="100" --mysql_instance_storage_size=1000 --additional_flags='"'"--cloud_storage_bucket=${cloud_storage_bucket}"'"')
+  run_uri=$(python perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py --run_stage=${prepare_provision} --mysql_svc_db_instance_cores=${mysql_svc_db_instance_cores} --mysql_svc_oltp_table_size="12000000" --mysql_svc_oltp_tables_count="100" --mysql_instance_storage_size=${mysql_instance_storage_size} --additional_flags='"'"--cloud_storage_bucket=${cloud_storage_bucket}"'"')
```

Comment:
here and below, there are still some constants being referred to by the bash command, like 12000000 or 100 or 1200, although all of those are defined as constants at the top of this script.

Is it hard to replace them because of the quotes? if so you can do this:

mysql_svc_oltp_tables_count=100

then in the bash invocation, you can do:

--mysql_svc_oltp_tables_count="{mysql_svc_oltp_tables_count}"

Would that work? try it with echo first.